src/vs/workbench/parts/terminal/electron-browser/terminalColorRegistry.ts

```diff
@@ -19,6 +19,13 @@ export const TERMINAL_FOREGROUND_COLOR = registerColor('terminal.foreground', {
 	dark: '#CCCCCC',
 	hc: '#FFFFFF'
 }, nls.localize('terminal.foreground', 'The foreground color of the terminal.'));
+export const TERMINAL_CURSOR_FOREGROUND_COLOR = registerColor('terminalCursor.foreground', {
+	light: '#333333',
```

Comment:
I think we should set the defaults to null and dynamically pull the inverted `terminal.background`/`.foreground` when they're null.